integration_test/third_party_apps_data/applications/jetty/metadata.yaml

```diff
@@ -0,0 +1,61 @@
+public_url: "https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/jetty"
+short_name: Jetty
+long_name: Jetty
+description: |-
+  The Jetty integration collects
+  access logs. Access logs are parsed into a JSON payload
+  focused on request details. 
+
+  For more information about Jetty, see
+  [The Jetty Docs](https://www.eclipse.org/jetty/documentation/jetty-11/operations-guide/index.html).
+minimum_supported_agent_version:
+  metrics: 2.9.0
+  logging: 2.9.0
+supported_app_version: ["11.x", "10.x", "9.4.x"]
+expected_logs:
```

Comment:
Can we also add the expected log for the `jetty_system` logs? See https://github.com/GoogleCloudPlatform/ops-agent/blob/230384c4fdd2a502439c6b8f172381e07a13079a/integration_test/third_party_apps_data/applications/activemq/metadata.yaml#L90 for an example for when an application sends logs to `syslog`